### ISSUE TITLE
Add a way to hide the weapon that is equipped by a ped / setPedWeaponVisible()

### ISSUE THREAD

**Issue by PlatinMTA** [multitheftauto/mtasa-blue#2364]
**Is your feature request related to a problem? Please describe.**
There should be a way to officially set the weapon of a ped visible or not. In GTA:SA there are certain cases where the weapon will be on hand but CJ wont show it (for example in Home Invasion, CJ will not show his weapon if he has a box in he's hand, even if he has its slot selected. This is exploited in speedruns.)

![image](https://user-images.githubusercontent.com/23291573/133912546-197d0fd3-c875-4c87-99f8-26a6b17ae5aa.png)
There's no weapon being shown, but it's equiped and being used.

**Describe the solution you'd like**
Having a function like setPedWeaponVisible, getPedWeaponVisible that goes:
```lua
setPedWeaponVisible( element player, bool visible )
getPedWeaponVisible( element player )
```

**Describe alternatives you've considered**
If this trick is done just by changing the alpha of the weapon, then it would be cool to have a function like ``setPedWeaponAlpha``.

**Additional context**
I already know that there are ways to do this with shaders, but I would argue that if there is an already coded function or "mta way" to do this, we should add it.

**Comment by ghost** [comment]
You can replace weapon model with empty model, or use LOD functions (didn't tested by myself). Although i agree that function like this should exist. Same goes with ability to disable weapon removal when there is no more ammo.

**Comment by Einheit-101** [comment]
Isnt this fixed by using engineSetModelLODDistance(weaponModelID, 0.001) and this new PR: https://github.com/multitheftauto/mtasa-blue/commit/efed59b7dc7b076219f1c8a868ef8aa028582127

**Comment by ghost** [comment]
> Isnt this fixed by using engineSetModelLODDistance(weaponModelID, 0.001) and this new PR: [efed59b](https://github.com/multitheftauto/mtasa-blue/commit/efed59b7dc7b076219f1c8a868ef8aa028582127)

Yes, you can use first function to selectively disable weapon models. setWeaponRenderEnabled disables rendering at all (no unnecessary code running under the hood)
